app/uk/gov/hmrc/plasticpackagingtax/registration/views/partner/partner_check_answers_page.scala.html

```diff
@@ -58,7 +58,7 @@
 @soleTraderSummary() = @{
     SummaryList(
         rows = Seq(
-            viewUtils.summaryListRow("partner.check.orgType", partner.partnerType.map(displayName(_)), None), // TODO: add change link
+            viewUtils.summaryListRow("partner.check.orgType", partner.partnerType.map(displayName(_)), Some(partnerRoutes.NominatedPartnerTypeController.displayExistingPartner(partner.id))), // TODO: add change link
```

Comment:
Remove the `TODO` since you've added the change link.

Reply:
Done